assets/i18n/qqq.json

```diff
@@ -346,6 +346,7 @@
 	"I18N_LEARNER_DASHBOARD_EMPTY_FEEDBACK_THREADS": "Text displayed in the learner dashboard. - This text is shown when there are no feedback threads to which the user is subscribed.",
 	"I18N_LEARNER_DASHBOARD_EMPTY_INCOMPLETE_COLLECTIONS": "Text displayed in the learner dashboard. - This text is shown when there is at least one collection in the complete section but no collections in the incomplete section of the user.",
 	"I18N_LEARNER_DASHBOARD_EMPTY_INCOMPLETE_EXPLORATIONS": "Text displayed in the learner dashboard. - This text is shown when there is at least one exploration in the complete section but no explorations in the incomplete section of the user.",
+	"I18N_LEARNER_DASHBOARD_EMPTY_LEARN_SOMETHING_NEW_SECTION": "Text displayed in the learner dashboard. - This text is shown when the user has reached the goal selection limit.",
```

Comment:
What does the "goal selection limit" mean? Can we explain this in slightly non-technical terms as people use this file to understand what the context for translating it is.

Reply:
Done!